### PR TITLE
Testing / 5.0

### PR DESCRIPTION
**Pull request by alexAubin**
Followup of https://github.com/YunoHost-Apps/peertube_ynh/pull/361

### CODE REVIEW

**Comment by alexAubin** [comment]
!testme (though not hugely necessary idk)

**Comment by fflorent** [comment]
!testme

File: scripts/install
Comment (fflorent) [inline scripts/install:38]: I have a slight doubt for this line (the configuration file rather suggests using an `openssl rand` command).

Looking at the code, I think that will be probably be fine.

Still, I submitted my question here: https://framacolibri.org/t/possible-value-for-secrets-peertube-in-production-yaml/16644

**Review by fflorent — COMMENTED** [review]
I still have a pending doubt (see my comment), otherwise LGTM.

Before merging, could you please consider merging #363 after the CI passed into testing?

File: scripts/install
Comment (fflorent) [reply scripts/install:38]: OK, that seems all good!

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**
